go.mod

```diff
@@ -9,13 +9,18 @@ require (
 )
 
 require (
+	github.com/g41797/sputnik v0.0.21
```

Comment:
we can not rely on this package since it has not wide community support and maintenance